packages/core/src/event_emitter.ts

```diff
@@ -105,53 +105,31 @@ class EventEmitter_ extends Subject<any> {
   }
 
   subscribe(generatorOrNext?: any, error?: any, complete?: any): Subscription {
-    let schedulerFn: (t: any) => any;
+    let schedulerFn = (t: any) => {};
```

Comment:
I think we can also avoid creating a noop function here and just state that `schedulerFn` can actually be undefined:
```suggestion
    let schedulerFn: ((t: any) => any)|undefined;
```
The `super.subscribe` function below accepts `undefined` as an argument.